gtsfm/utils/io.py

```diff
@@ -271,8 +275,11 @@ def read_cameras_txt(fpath: str) -> Optional[List[Cal3Bundler]]:
                 float(k2),
             )
             calibrations.append(Cal3Bundler(fx, k1, k2, u0, v0))
-
+            if return_img_dims:
+                img_dims.append((img_h, img_w))
     assert len(calibrations) == num_cams
+    if return_img_dims:
+        return calibrations, img_dims
```

Comment:
nit: Can we always return a tuple of calibrations and img_dims? I've found that a variable number of return arguments can lead to some bugs and I find the signature a bit hard to read.

Reply:
Marking as resolved, thanks!